docs/ops/sequence/AUGRUSequence_10.md

```diff
@@ -0,0 +1,145 @@
+# AUGRUSequence  {#openvino_docs_ops_sequence_AUGRUSequence_10}
+
+**Versioned name**: *AUGRUSequence-10*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUSequence* operation represents a series of AUGRU cells (GRU with attentional update gate).
+
+**Detailed description**: The difference between *AUGRUSequence* and [GRUSequence](./GRUSequence_5.md) is the additional attention score input `A`, which is a multiplier for the update gate.
+The formula was described in the [paper arXiv:1809.03672](https://arxiv.org/abs/1809.03672).
+
+The sequence can be connected differently depending on `direction` attribute that specifies the direction of traversing of input data along sequence dimension or specifies whether it should be a bidirectional sequence.
+
+The most of the attributes are in sync with the specification <a href="https://github.com/onnx/onnx/blob/master/docs/Operators.md#gru">ONNX GRU</a> operator.
+
```

Comment:
@mitruska, it would be really nice if you add calculation scheme for this operation. There can be several versions of `AUGRU` so we can avoid confusion in the implementation.